sql/btree_indexes_on_array_columns.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-sql
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+-- Finds B-tree indexes on array columns
+--
+-- GIN-index should be used instead for such columns
+-- Based on query from https://habr.com/ru/articles/800121/
+select
+    i.indrelid::regclass as table_name,  -- Name of the table
+    i.indexrelid::regclass as index_name, -- Name of the index
+    a.amname, -- Index type
```

Comment:
I'm not sure that we need `amname` here because it's constant

Reply:
Thanks. Yes, it's always `btree`

Removed it from query